scripts/submitHistoFiller_lxplus.py

```diff
@@ -0,0 +1,147 @@
+import os
+import sys
+import argparse
+
+
+write = lambda stream, text: stream.write(text + '\n')
+
+
+def make_parser():
+    parser = argparse.ArgumentParser(description='Command line parser of plotting options')
+    parser.add_argument('--outdir', type=str, dest="outdir", required=True,
+                        help='/afs/directory/to/submit/from. Must be afs\
+    because condor submission is only allowed from /afs on lxplus.')
+    parser.add_argument('--cfg', dest='cfg', required=True,
+                        help='name of the main cfg to run', default=None)
+    parser.add_argument('--tag', dest='tag', required=True, default=None,
+                        help='name of working space')
+    parser.add_argument('--njobs', dest='njobs', required=False, type=int,
+                        help='number of jobs for parallelization', default=10)
+    return parser
+
+
+def create_dir(d):
+    if not os.path.exists(d):
+        print("{} does not exist. Creating now".format(d))
+        os.makedirs(d)
+
+
+def write_filler_executable(scriptpath, exe, cfg, njobs):
+    with open(scriptpath, 'w') as s:
+        write(s,'#!/bin/bash')
+        write(s,'export X509_USER_PROXY=~/Condor/x509up_u149480')
+        write(s,'source /cvmfs/cms.cern.ch/cmsset_default.sh')
+        write(s,'cd {} || exit 1'.format(os.getcwd()))
+        write(s,'export SCRAM_ARCH=slc6_amd64_gcc491')
```

Comment:
probably not super important, but we should maybe update this to `slc7_amd64_gcc820` (also the case for the submission script at LLR)

Reply:
I actually forgot about this one for this commit. Since it's not super important i'll change it in the next commit.